Gemfile.lock

```diff
@@ -266,125 +268,127 @@ GEM
       minitest (>= 5.1)
       tzinfo (~> 2.0)
       zeitwerk (~> 2.3)
-    addressable (2.8.4)
+    addressable (2.8.6)
       public_suffix (>= 2.0.2, < 6.0)
-    airbrake (11.0.1)
+    airbrake (11.0.3)
       airbrake-ruby (~> 5.1)
-    airbrake-ruby (5.2.0)
+    airbrake-ruby (5.2.1)
       rbtree3 (~> 0.5)
     ansible (0.2.2)
     ar_multi_threaded_transactional_tests (0.5.0)
       activerecord (>= 4.2.0, < 6.2.0)
-    ast (2.4.1)
+    ast (2.4.2)
```

Comment:
### [ast](https://whitequark.github.io/ast/)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.4.1 → 2.4.2

[_~~change-log~~, [source-code](https://github.com/whitequark/ast), [gem-diff](https://my.diffend.io/gems/ast/2.4.1/2.4.2)_]





<details>
<summary>Commits</summary>

A change of **5** commits. See the full changes on [the compare page](https://github.com/whitequark/ast/compare/v2.4.1...v2.4.2).

These are the individual commits:
- (2a878a2) [Add Truffleruby head to CI](https://github.com/whitequark/ast/commit/2a878a2b5fb33524b03e9d5cd176a4980ed21a8a)
- (d9ea9fe) [Enable matching with `Node#deconstruct` (#31)](https://github.com/whitequark/ast/commit/d9ea9fea92dfc7a53528e89e2c50f2a03b465194)
- (fa14f33) [added dummy test github action](https://github.com/whitequark/ast/commit/fa14f33cbe90812728f8de35ac4fdb771a3f3558)
- (0b02fcf) [Run tests using github actions (#32)](https://github.com/whitequark/ast/commit/0b02fcf49659d7aada2d5246c2f84e9d2cedaef8)
- (272f4a8) [Bump version.](https://github.com/whitequark/ast/commit/272f4a883573dacf0a33b4799e38e89e0b4c7dc8)


</details>